javacorelib/src/main/scala/java/lang/Thread.scala

```diff
@@ -72,7 +72,7 @@ object Thread {
   }
 
   def sleep(millis: scala.Long, nanos: scala.Int): Unit = {
-    import scala.scalanative.posix.errno.EINTR
+    /*import scala.scalanative.posix.errno.EINTR
```

Comment:
This needs to be refactored by moving the implementation to `runtime.Platform` as well.

Reply:
Yes, I am fighting with `javafilelib` at the moment, after that I will fix all this small issues about `System.scala`

Reply:
I made `platform.cpp` file and there I made `thread_sleep` method.
The reason why I made .cpp file because Microsoft stopped to support C11 etc headers, it only works in C++ now.